src/bundle/Resources/public/scss/_add-translation.scss

```diff
@@ -7,15 +7,28 @@
             display: inline-block;
             left: 50%;
             transform: translateX(-50%);
+            border-radius: .25rem;
         }
     }
 
     .modal-header {
-        background: $ez-ground-base;
+        background: $ez-color-base-pale;
+        border-bottom: none;
+
+        .close {
+            margin-top: -.55rem;
```

Comment:
If the root font size is `16px` then `.55rem` is `8.8px`. Maybe it should be `.5625rem` (`9px`) to make it rounded to the nearest number.